### ISSUE TITLE
hyperparameter settings request,or training log

### ISSUE THREAD

**Issue by QiyaoHuang** [RingBDStack/SUGAR#4]
Thanks for answering my last question,but our team still didn't get the performance in the paper.Can we get the hyperparameter settings in MuTag,NCI1,PTC_MR,Protein.or your training logs in these datasets?

**Comment by Suchun-sv** [comment]
Training Log might not exists anymore, Let me tried to recover exp statics to find best hyperparameter setting. It may take time, we are busy in ddls. And if you have interests, you can use hyperparamter tuna framework like optnua to get good metrics.

**Comment by Suchun-sv** [comment]
May I ask what your team's goal for using this repo, this could instruct us to help your team.

**Comment by QiyaoHuang** [comment]
> May I ask what your team's goal for using this repo, this could instruct us to help your team.
The reviewer required us to compare your model in our paper because your model works well.

**Comment by Suchun-sv** [comment]
We use the optuna framework (https://optuna.org/) for hyperparameter tuning on each dataset. 
The inconsistent performance may also be due to different dataset partitions. If you want to use it as the baseline, you can directly report the results running under your dataset partition or tune it using optuna.

**Comment by Suchun-sv** [comment]
We use the optuna framework (https://optuna.org/) for hyperparameter tuning on each dataset. The tuning range is 1e-3 to 1e-5 for lr, 0.5 to 1. for MI_loss, 300 to 1000 for epochs.
The inconsistent performance may also be due to different dataset partitions. If you want to use it as the baseline, you can directly report the results running under your dataset partition or tune it using optuna.

**Comment by QiyaoHuang** [comment]
> We use the optuna framework (https://optuna.org/) for hyperparameter tuning on each dataset. The tuning range is 1e-3 to 1e-5 for lr, 0.5 to 1. for MI_loss, 300 to 1000 for epochs. The inconsistent performance may also be due to different dataset partitions. If you want to use it as the baseline, you can directly report the results running under your dataset partition or tune it using optuna.

We use the default hyperparameter in your project,but the results of training are not satisfactory in datasets, such like this:
PRT_MR
folds 1/10: 100%|██████████| 1000/1000 [59:00<00:00,  3.54s/it, k:0.80, loss: 0.56, best_acc:0.50, RL:309]
folds 2/10: 100%|██████████| 1000/1000 [58:52<00:00,  3.53s/it, k:0.85, loss: 0.56, best_acc:0.38, RL:330]
folds 3/10: 100%|██████████| 1000/1000 [53:45<00:00,  3.23s/it, k:0.91, loss: 0.55, best_acc:0.62, RL:310]
folds 4/10: 100%|██████████| 1000/1000 [49:56<00:00,  3.00s/it, k:0.88, loss: 0.56, best_acc:0.56, RL:313]
folds 5/10: 100%|██████████| 1000/1000 [49:39<00:00,  2.98s/it, k:0.99, loss: 0.67, best_acc:0.65, RL:299]
folds 6/10: 100%|██████████| 1000/1000 [50:18<00:00,  3.02s/it, k:0.93, loss: 0.56, best_acc:0.71, RL:299]
folds 7/10: 100%|██████████| 1000/1000 [50:20<00:00,  3.02s/it, k:0.95, loss: 0.56, best_acc:0.82, RL:299]
folds 8/10: 100%|██████████| 1000/1000 [50:37<00:00,  3.04s/it, k:0.86, loss: 0.56, best_acc:0.65, RL:333]
folds 9/10: 100%|██████████| 1000/1000 [49:00<00:00,  2.94s/it, k:0.90, loss: 0.56, best_acc:0.76, RL:322]
folds 10/10: 100%|██████████| 1000/1000 [48:46<00:00,  2.93s/it, k:0.85, loss: 0.56, best_acc:0.59, RL:338]

and the default hyperparameter :
parser.add_argument('--dataset', type=str, default="MUTAG")
# parser.add_argument('--num_info', type=float, default=1)
parser.add_argument('--lr', type=float, default=0.001)
parser.add_argument('--max_pool', type=float, default=0.06)
parser.add_argument('--momentum', type=float, default=0.8)
parser.add_argument('--num_epoch', type=int, default=1000)
parser.add_argument('--batch_size', type=int, default=256)
parser.add_argument('--sg_encoder', type=str, default='GCN')
parser.add_argument('--MI_loss', type=float, default=0.8)
parser.add_argument('--start_k', type=float, default=0.8)
